### ISSUE TITLE
Gadgetsmenu Permissions issue [HELP]

### ISSUE THREAD

**Issue by Javi842** [yapzhenyie/GadgetsMenu#847]
Hi, i have a server with gadgetsmenu plugin, i want to put some cosmetics only for people with purchased ranks, but its impossible idk why, first of all, everybody can buy every cosmetic avaible no matter if he/she have the permission, the only thing that happens is when the person buy a cosmetic (exclusive for rank) it shows the message that he/she doesnt have the permission to dot that BUT nevertheless the following message is that he/she have bought the cosmetic successfully and they get it, idk why is this happening,I think that maybe it is this option in the config.yml:   

""# This is the command when player purchase cosmetic items.""
""# This command is depends on your permission plugin.""
 ""Execute-Command: lp user {PLAYER} permission set {PERMISSION}""

BUT, When i disable the execute-command putting nothing or just "[]", no one can purchase a cosmetic, and it gaves me in the console this error:

https://paste.shockbyte.com/elagikotuzenidizadif

IMPORTANT:
It should be noted that I have removed and set the permissions for users with ranks and for users without rank in my plugin of permissions.

In conclusion: I want to make some cosmetics exclusive for ranks and some other totally free, how i make that possible taking into account what I have done.
